inst/tutorials/introduction/introduction.Rmd

```diff
@@ -0,0 +1,265 @@
+---
+title: "malariasimulation: an introductory tutorial"
+output: learnr::tutorial
+runtime: shiny_prerendered
+---
+
+```{r setup, include=FALSE}
+library(learnr)
+library(malariasimulation)
+library(dplyr)
+library(ggplot2)
+library(tidyr)
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+
+# `malariasimulation` overview
+
+`malariasimulation` in an individual based malaria model. This tutorial is an entry-level guide to using the model package in R.
+
+Use of `malariasimulation` can be divided into four stages:
+
+
+![](images/malsim_overview.png){width=100%}
```

Comment:
I'm wondering how common "wrangling" is? Would "processing outputs" be more clear for lots of different language speakers?